h2o-py/h2o/automl/_base.py

```diff
@@ -125,3 +125,72 @@ def event_log(self):
         :return: an H2OFrame with detailed events occurred during the AutoML training.
         """
         pass
+
+    def get_best_model(self, algorithm, criterion=None):
```

Comment:
Why making `algorithm` a mandatory parameter.
It is completely legitimate to ask for the best model accroding to a given criterion, regardless of the algo.

Reply:
Thank you, I missed that use-case.